modules/importer/src/runner/common/walker/git.rs

```diff
@@ -202,7 +202,8 @@ where
                 builder.branch(branch);
             }
 
-            let fo = Self::create_fetch_options();
+            let mut fo = Self::create_fetch_options();
+            fo.depth(2);
```

Comment:
Ok, that might cause a problem later on in the code. In case of a continuation token (the git commit sha), having a limited depth will break that logic: https://github.com/trustification/trustify/blob/f66ef06d635a81f44c8e696b191b2fcc5144e6dd/modules/importer/src/runner/common/walker/git.rs#L249-L288